scalanativelib/src/ScalaNativeModule.scala

```diff
@@ -77,7 +85,7 @@ trait ScalaNativeModule extends ScalaModule { outer =>
 
   def bridgeFullClassPath = T {
     Lib.resolveDependencies(
-      Seq(coursier.LocalRepositories.ivy2Local, MavenRepository("https://repo1.maven.org/maven2")),
+      repositories = repositoriesTask(),
       Lib.depToDependency(_, scalaVersion(), platformSuffix()),
       toolsIvyDeps(),
       ctx = Some(implicitly[mill.util.Ctx.Log])
```

Comment:
This can be replaced by `resolveDeps(toolsIvyDeps())`.

Reply:
Done ✅

Reply:
I had to revert the change because then it was resolving scala-library 2.13 instead of 2.12 (which is the correct one for tools).
My best guess is that it resolves the wrong one because of this https://github.com/com-lihaoyi/mill/issues/860
Added a comment to explain the situation.